cmd/switch.go

```diff
@@ -15,55 +15,117 @@
 package cmd
 
 import (
+	"fmt"
+
 	"github.com/spf13/cobra"
 
 	"github.com/tetratelabs/getistio/api"
 	"github.com/tetratelabs/getistio/src/istioctl"
 	"github.com/tetratelabs/getistio/src/util/logger"
 )
 
+type switchFlags struct {
+	name, version, flavor string
+	flavorVersion         int64
+}
+
 func newSwitchCmd(homedir string) *cobra.Command {
-	var (
-		flagVersion       string
-		flagFlavor        string
-		flagFlavorVersion int
-	)
+	var flag switchFlags
 
 	cmd := &cobra.Command{
-		Use:   "switch <istio version>",
+		Use:   "switch",
 		Short: "Switch the active istioctl to a specified version",
 		Long:  `Switch the active istioctl to a specified version`,
-		Example: `# switch the active istioctl version to version=1.7.4, flavor=tetrate and flavor-version=1
-$ getistio switch --version 1.7.4 --flavor tetrate --flavor-version=1`,
+		Example: `# Switch the active istioctl version to version=1.7.4, flavor=tetrate and flavor-version=1
```

Comment:
Just want to confirm that you verified that these examples actually works, right? I cannot check right now because I'm not in front of my pc.

Reply:
checked on my side